app/views/embed/_analytics.html.erb

```diff
@@ -16,3 +16,20 @@
   }
   gtag('config', 'G-80J719XWE5', args)
 </script>
+<script src="https://unpkg.com/ahoy.js"></script>
```

Comment:
Can we pin to a specific version?  Should we add an integrity attribute https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity ?

Reply:
Both good suggestions! Added.